controllers/property_controller/property_address/updatePropertyAddress.js

```diff
@@ -80,6 +80,10 @@ module.exports = {
             trim: true,
             isString: true,
             escape: true,
+            isLength: {
+                errorMessage: 'zipcode should be at least 3 characters long',
+                options: { min: 3 },
```

Comment:
3? I thought zip codes have to have 5 digits?

Reply:
Depends, right now this just makes sure something is there, but postal code varies, maybe not so much in the US, but if we consider growing to foreign markets they can be all types of lengths. One future task would be to just validate that it's an actual valid postal code using some pre existing API.